### PR TITLE
Testing/dashboard

### PR DESCRIPTION
**Pull request by kevinhartmann23**
## *Purpose:*

- [x] Adds a new feature
- [ ] Improves an existing feature
- [ ] Fixes a bug
- [x] Cleans up (logs, tests, etc.)

## *What are the relevant tickets?*
 
Closes _

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have used tests to prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my change

### CODE REVIEW

File: cypress/fixtures/imageData.json [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+          "title": "pavement"
+        },
+        {
+          "type": "search",
+          "title": "sidewalk"
+        }
+      ]
+    }
+  ]
+}
```
Comment (srslie) [inline cypress/fixtures/imageData.json:163]: Great sample set.

File: cypress/integration/dashboard.spec.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  it('should display four option cards to view certain color hues', () => {
+    cy
+      .visit(baseUrl)
+      .get('.Red')
+      .get('.card-header').eq(0).should('have.text', 'Red')
+      .get('.card-header').eq(1).should('have.text', 'Green')
+      .get('.card-header').eq(2).should('have.text', 'Blue')
+      .get('.card-header').eq(3).should('have.text', 'B & W')
+  })
+
```
Comment (srslie) [inline cypress/integration/dashboard.spec.js:14]: Nice use of intercept.

File: cypress/integration/resultsDisplay.spec.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    cy
+      .visit(baseUrl)
+      .get('.Red-button').click()
+
+  })
+
+  it('should practice RESTful, each url should have search term in path', () => {
+    cy
+      .on("url:changed", newUrl => {
+        expect(newUrl).to.contain('movies/694919')
```
Comment (srslie) [inline cypress/integration/resultsDisplay.spec.js:18]: Cool syntax, that's new to me.

**Review by srslie — APPROVED** [review]
Thank you!

**Comment by christina-cook** [comment]
- [ ] Changes generate no new warnings or errors
- [ ] Branch was tested locally without failures
- [x] All tests are passing as expected

File: cypress/integration/resultsDisplay.spec.js (hunk 3 above)
Comment (christina-cook) [reply cypress/integration/resultsDisplay.spec.js:18]: Oops :)

**Pull request merged**